src/supported_apps.ts

```diff
@@ -277,4 +277,16 @@ export const supportedApps: SubstrateAppParams[] = [
     slip0044: 0x80000219,
     ss58_addr_type: 126,
   },
+  {
+    name: "Enjin",
+    cla: 0xb9,
+    slip0044: 0x80000483,
+    ss58_addr_type: 2135,
+  },
+  {
+    name: "Matrixchain",
+    cla: 0xef,
```

Comment:
Could `0xba` CLA be used?

Reply:
O chose ef because it is exactly our ss58 address prefix, all addresses start with ef, so it would be special for us 😄 But if you think that will be too much throuble I can changr

Reply:
This is something internal for ledger, I'm not sure what happens on that range of CLAs. Better to stick to 0xba